MicroQRArt/ViewController/Profile/ProfileViewController.swift

```diff
@@ -6,10 +6,144 @@
 //
 
 import UIKit
+import ReactiveSwift
+import ReactiveCocoa
 
 final class ProfileViewController: UIViewController {
+    
+    // MARK: - Properties
+    private let viewModel: MyQRCodeListViewModel
+    private let disposables = CompositeDisposable()
+    private let profileView = ProfileView()
+
+    // MARK: - Reactive Properties
+    private let deleteActionPipe = Signal<IndexPath, Never>.pipe()
+    private let editActionPipe = Signal<(IndexPath, String), Never>.pipe()
+    private let viewDidLoadPipe = Signal<Void, Never>.pipe()
+    private let editAlertRequestPipe = Signal<IndexPath, Never>.pipe()
+
+    // MARK: - Data Source
+    private lazy var dataSource = MyQRCodeTableViewDataSource(
+        animationConfig: AnimationConfiguration(
+            insertAnimation: .automatic,
+            deleteAnimation: .automatic,
+            reloadAnimation: .automatic
+        )
+    )
+
+    // MARK: - Initialization
+    init(viewModel: MyQRCodeListViewModel = MyQRCodeListViewModel()) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    deinit {
+        disposables.dispose()
+    }
+
+    // MARK: - Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .green
+        setupView()
+        bindViewModel()
+        viewDidLoadPipe.input.send(value: ())
+    }
+
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        profileView.frame = view.bounds
+    }
+
+    // MARK: - Setup
+    private func setupView() {
+        profileView.frame = view.bounds
+        view.addSubview(profileView)
+        
+        profileView.tableView.delegate = self
+        profileView.tableView.dataSource = dataSource
+    }
+
+    private func bindViewModel() {
```

Comment:
@donuts-yoshidahikaru 
細かいですが、統一性のため、ここはsetupBindでお願いします。
```suggestion
    private func setupBind() {
```

Reply:
@donuts-wuchunyen 

6383de436595795e908cd1f604f09b7d9414eba3 で対応いたしました。